src/lib/broadcast.jl

```diff
@@ -179,7 +179,13 @@ end
 end
 
 @init @require CuArrays="3a865a2d-5b23-5a0f-bc46-62713ec82fae" begin
-  @adjoint function broadcasted(::Broadcast.ArrayStyle{CuArrays.CuArray}, f, args...)
+  if isdefined(CuArrays, :CuArrayStyle)  # Introduced in CuArrays v2.0
```

Comment:
Since we claim compliance with CuArrays 2.x do we need to check for this anymore? Old versions of flux and Zygote would just never get new CuArrays, and the new ones wouldn't get old CuArrays, right?

Reply:
> Since we claim compliance with CuArrays 2.x

Where do we do that? I didn't think Zygote had a direct dependency on CuArrays.